### PR TITLE
Sync "6.0/stage" with "master" via GitHub Actions

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

File: .github/scripts/sync-with-master.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# We need the full git repository history in order to do the git-merge.
+#
+git fetch --unshallow
+git merge origin/master
+
+#
+# We modify the local branch name and upstream tracking branch such that
+# the correct operations occur when pushing and opening the pull request.
+#
+git branch -M "sync/${BRANCH}"
```
Comment (sdimitro) [inline .github/scripts/sync-with-master.sh:28]: not a big deal but maybe change this to `sync-with-master/${BRANCH}` or something like that?

**Comment by sdimitro** [comment]
@prakashsurya do you have a run of this handy for us to examine the output/log of this job?

**Comment by prakashsurya** [comment]
@sdimitro try this: https://github.com/prakashsurya/sdb/runs/610037951 .. which opened this PR: https://github.com/prakashsurya/sdb/pull/11

File: .github/scripts/sync-with-master.sh (hunk 1 above)
Comment (prakashsurya) [reply .github/scripts/sync-with-master.sh:28]: done.

**Pull request merged**